src/file/SDL_iostream.c

```diff
@@ -741,12 +741,16 @@ SDL_IOStatus SDL_GetIOStatus(SDL_IOStream *context)
 }
 
 
-SDL_IOStream *SDL_OpenIO(const SDL_IOStreamInterface *iface, void *userdata)
+SDL_IOStream *SDL_OpenIO(const SDL_IOStreamInterface *iface, int version, void *userdata)
 {
     if (!iface) {
         SDL_InvalidParamError("iface");
         return NULL;
     }
+    if (version != SDL_IOSTREAM_INTERFACE_VERSION) {
+        SDL_SetError("Unknown interface version");
+        return NULL;
+    }
```

Comment:
Maybe make this a switch statement to imply your _possible_ intent to handle things differently with multiple known versions in future ??

Reply:
Sure, that sounds good.